client/src/components/PeriodicTable/ElementInfo.js

```diff
@@ -0,0 +1,831 @@
+        elementType: ElementTypes.metal
+    },
+    hafnium: {
+        atomicNumber: 72,
+        atomicMass: 178.49,
+        elementName: "Hafnium",
+        elementSymbol: "Hf",
+        elementType: ElementTypes.metal
+    },
+    tantalum: {
+        atomicNumber: 73,
+        atomicMass: 180.9479,
+        elementName: "Tantalum",
+        elementSymbol: "Ta",
+        elementType: ElementTypes.metal
+    },
+    tungsten: {
+        atomicNumber: 74,
+        atomicMass: 183.85,
+        elementName: "Tungsten",
+        elementSymbol: "W",
+        elementType: ElementTypes.metal
+    },
+    rhenium: {
+        atomicNumber: 75,
+        atomicMass: 186.207,
+        elementName: "Rhenium",
+        elementSymbol: "Re",
+        elementType: ElementTypes.metal
+    },
+    osmium: {
+        atomicNumber: 76,
+        atomicMass: 190.2,
+        elementName: "Osmium",
+        elementSymbol: "Os",
+        elementType: ElementTypes.metal
+    },
+    iridium: {
+        atomicNumber: 77,
+        atomicMass: 192.22,
+        elementName: "Iridium",
+        elementSymbol: "Ir",
+        elementType: ElementTypes.metal
+    },
+    platinum: {
+        atomicNumber: 78,
+        atomicMass: 195.09,
+        elementName: "Platinum",
+        elementSymbol: "Pt",
+        elementType: ElementTypes.metal
+    },
+    gold: {
+        atomicNumber: 79,
+        atomicMass: 196.9665,
+        elementName: "Gold",
+        elementSymbol: "Au",
+        elementType: ElementTypes.metal
+    },
+    mercury: {
+        atomicNumber: 80,
+        atomicMass: 200.59,
+        elementName: "Mercury",
+        elementSymbol: "Hg",
+        elementType: ElementTypes.metal
+    },
+    thallium: {
+        atomicNumber: 81,
+        atomicMass: 204.37,
+        elementName: "Thallium",
+        elementSymbol: "Tl",
+        elementType: ElementTypes.metal
+    },
+    lead: {
+        atomicNumber: 82,
+        atomicMass: 207.2,
+        elementName: "Lead",
+        elementSymbol: "Pb",
+        elementType: ElementTypes.metal
+    },
+    bismuth: {
+        atomicNumber: 83,
+        atomicMass: 208.9804,
+        elementName: "Bismuth",
+        elementSymbol: "Bi",
+        elementType: ElementTypes.metal
+    },
+    polonium: {
+        atomicNumber: 84,
+        atomicMass: 209,
+        elementName: "Polonium",
+        elementSymbol: "Po",
+        elementType: ElementTypes.metalloid
+    },
+    astatine: {
+        atomicNumber: 85,
+        atomicMass: 210,
+        elementName: "Astatine",
+        elementSymbol: "At",
+        elementType: ElementTypes.metalloid
+    },
+    radon: {
+        atomicNumber: 86,
+        atomicMass: 222,
+        elementName: "Radon",
+        elementSymbol: "Rn",
+        elementType: ElementTypes.metalloid
+    },
+    francium: {
+        atomicNumber: 87,
+        atomicMass: 223,
+        elementName: "Francium",
+        elementSymbol: "Fr",
+        elementType: ElementTypes.metalloid
```

Comment:
I believe 87 Francium should be a metal if I'm not wrong. Double check the element types!

Reply:
https://www.angelo.edu/faculty/kboudrea/periodic/physical_metals.htm